### DISCUSSION TITLE
Order global search results like $navigationSort enables ordering in the navbar

### DISCUSSION THREAD

**Discussion by lucacastelnuovo** [filamentphp/filament#5618] [Help] [answered]
Hi all, Great Package!

Ordering items in the navbar is possible by setting the variable `$navigationSort`.
Can you also specify this order by a different method for the global search results?

**Comment by lucacastelnuovo** [comment]
P.S. Is there also a possibility to limit of search results returned?

This code does not have any effect.
```php
protected static function getGlobalSearchEloquentQuery(): Builder
    {
        return parent::getGlobalSearchEloquentQuery()
            ->limit(2);
    }
```

**Reply by ArtMin96** [reply, marked as answer]
```php
public static function getGlobalSearchResultsLimit(): int
{
    return 10;
}
```

**Reply by lucacastelnuovo** [reply]
Thank you for this answer. Do you also know how to solve the other problem?

**Comment by ArtMin96** [comment]
You can reach that with an eloquent query: [https://filamentphp.com/docs/2.x/admin/resources/global-search#details](https://filamentphp.com/docs/2.x/admin/resources/global-search#details)

```php
protected static function getGlobalSearchEloquentQuery(): Builder
{
    return parent::getGlobalSearchEloquentQuery()->orderBy('column', 'sort');
}
```

**Reply by lucacastelnuovo** [reply]
Apologies if the question was not clear enough. I was talking about sorting the different types of models/resources that come up when enabling global search on multiple resources.

Like ordering resources in the navigation menu by moving (e.g. Users) higher in the menu than a different resource (e.g. Assets).

**Reply by raobilal482** [reply]
i a also searching about this problem if you solved then please describe how
